File: backend/components/metiers/administration_publique/utils/ai/samples/sample_ai_helper.test.js
// sample_ai_helper.test.js – Test unitaire ultra avancé sample helper IA (JS)
const { sampleAiHelper } = require('./sample_ai_helper');
test('sampleAiHelper retourne SAMPLE pour input', () => {
  const res = sampleAiHelper('foo');
  expect(res.status).toBe('SAMPLE');
  expect(res.data).toBe('foo');
  expect(res.audit).toBe(true);
});
test('sampleAiHelper retourne ERROR si pas d\'input', () => {
  const res = sampleAiHelper();
  expect(res.status).toBe('ERROR');
});

File: backend/components/metiers/administration_publique/utils/ai/samples/sample_ai_helper.test.py
# sample_ai_helper.test.py – Test unitaire ultra avancé sample helper IA (Python)
from .sample_ai_helper import sample_ai_helper

def test_sample_ai_helper_ok():
    res = sample_ai_helper('foo')
    assert res['status'] == 'SAMPLE'
    assert res['data'] == 'foo'
    assert res['audit'] is True

def test_sample_ai_helper_error():
    res = sample_ai_helper(None)
    assert res['status'] == 'ERROR'
